qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -239,25 +242,25 @@ def _build_schedules(self, basis_gates: Set[str]) -> Dict[str, ScheduleBlock]:
         dur = Parameter("duration")
         sigma = Parameter("σ")
 
-        x_amp, x_beta = Parameter("amp"), Parameter("β")
+        x_amp, x_beta, x_angle = Parameter("amp"), Parameter("β"), Parameter("angle")
 
         if self._link_parameters:
-            y_amp, y_beta = 1.0j * x_amp, x_beta
+            y_amp, y_beta, y_angle = x_amp, x_beta, x_angle + np.pi / 2
```

Comment:
I think angle must be separated to cope with mixer skew, i.e. angle difference is not exactly pi/2. So they should have independent angles but default value can be angle + pi/2 for y gate. Please confirm @wshanks

Reply:
We could do that, but I think that would contradict the logic which was in place before. If the amplitudes were linked, then the angles were also linked in the previous implementation. Linking the angles will preserve the previous logic. This doesn't mean that the previous logic is bullet-proof. What do you think @wshanks ?

Reply:
That's fair point. Since (sx, rz, cx) are enough to compose a universal set, I think experimentalist who requested sy to the library wants to correct for the mixer skew. But we can keep the logical change minimum and do this in the follow up.

Reply:
I think this is fine. The user has the option to unlink the parameters and therefore has full flexibility. We could be more granular as to what parameters we link and do not link but I would not go down that rabbit hole here.